pkgs/tools/filesystems/gocryptfs/default.nix

```
@@ -19,7 +19,7 @@ buildGoPackage rec {
     owner = "rfjakob";
     repo = "gocryptfs";
     rev = version;
```

```suggestion
    rev = "v${version}";
```

`version` won't have a `v` in it anymore.

---

Resolved with d5363cbd5ae94539f339da48a4e36dab64e0f181